woocommerce/google-listings-and-ads
# Strip shortcodes used in product description before sync

### Changes proposed in this Pull Request:



Closes #784.

This PR removes the shortcodes from the product's description before syncing the product to Google Merchant Center.


### Detailed test instructions:

1. Use shortcodes in a product's description (the shortcodes must be valid and existing, i.e. it doesn't remove anything within a bracket). For example, you can use `[video]` or `[gallery]` shortcodes.
2. Sync the product
3. Make sure the shortcodes are NOT present in the product description when viewing it on Google Merchant Center


### Changelog entry

Fix - Strip shortcodes used in the product description when syncing to Google
